src/static/scripts/ajaxify.js

```diff
@@ -0,0 +1,41 @@
+/* global ajax */
+/* exported loadContent */
+
+var loadContent = (function() {
+    var pageCache = {};
+
+    function createRequest(url, callback) {
+        function errorCallback() {
+            console.log('error');
+        }
+        function httpErrorCallback() {
+            console.log('http error');
+        }
+        function successCallback() {
+            console.log('success');
+            callback(this.responseText);
+        }
+        ajax(url)
+            .error(errorCallback)
+            .httpError(httpErrorCallback)
+            .success(successCallback)
+            .requestHeader('X-Ajax-Request', true)
```

Comment:
No, I mean the header that gets set automatically in most cases. Seems to be recommended in several places, only primary worry mentioned is that it can be spoofed, which is not a concern here.

But not important, just was wondering why you didn't use it.
https://guides.github.com/features/mastering-markdown/